docs/adv-topics/noderesolution.md

```diff
@@ -0,0 +1,47 @@
+# Node Resolution
+app_rpt has three ways to resolve a node number to an IP address.  
```

Comment:
Add a blank line after this sentence to separate the text from the list.

Reply:
I apologize for not making my comment clear.  I was asking for a blank line between the "app_rpt has three ways ..." and the first "* An external node ...".  Without the break the list does not show up as a list.